Sources/Plasma/NucleusLib/pnUtils/pnUtCrypt.cpp

```diff
@@ -95,7 +95,11 @@ static void Rc4Codec (
 ) {
     // RC4 uses the same algorithm to both encrypt and decrypt
     uint8_t * temp = (uint8_t *)malloc(bytes);
+
+    IGNORE_WARNINGS_BEGIN("deprecated-declarations")
```

Comment:
The changes in this file probably belong in another PR

Reply:
Split to #1476